packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -1025,6 +1056,21 @@ EncoderProfiles getRecordingProfile() {
     return cameraFeatures.getResolution().getRecordingProfile();
   }
 
+  Integer getFps() {
```

Comment:
Please add nullability annotations to all of these return values; we're generally adding them in new code.

Reply:
added `@Nullable`